src/ServiceLogger.js

```diff
@@ -203,6 +206,17 @@ class ServiceLogger {
         // move an object to the msg_json field that can be index by ELK
         // do nothing if dealing with a string
         // it is important to run this step before the message touches bunyan
+
+        const {child, wrapped} = additionalFields;
+
+        if (this.caller) {
+          const index = 3 + (child ? 1 : 0) + (wrapped ? 1: 0);
+          additionalFields.caller = new Error().stack.split("\n")[index].trim();
```

Comment:
I am not sure if this has an impact on performance e.g. if it happens at 500 logs per second

Reply:
also this feels like we should wrap it with a try/catch block; the building of that index and the fluent calls look rather dangerous :)